src/MOL-SLLOD/fix_nvt_asllod_mol.cpp

```diff
@@ -0,0 +1,305 @@
+// clang-format off
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/
+   Steve Plimpton, sjplimp@sandia.gov, Sandia National Laboratories
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Emily Kahl (Uni of QLD)
```

Comment:
Should update the "contributing author" since I didn't really write this one.

Reply:
Good point. Maybe we do a pass over all of that at the end though to make sure we have all the relevant people on each file?